SwiftWisdom/Core/StandardLibrary/Numbers/UnsignedInteger+Extensions.swift

```diff
@@ -50,7 +72,9 @@ extension UnsignedInteger {
         return NSData(bytes: &copy, length: MemoryLayout<Self>.size)
     }
 
-    /// Converts a bit mask into its given indexes. For example, `0b101` will return `[0,2]`
+    /// Converts a bit mask into its given indexes. For example, `0b10011` will return `[0, 1, 4]`
+    
+    //Converts a bit mast into it's given indexs
```

Comment:
Typos: "mast" -> "mask", "it's" -> "its", and "indexs" -> "indexes" (or "indices," but we use "indexes" on line 75.